demo/src/app/components/+accordion/demos/custom-html/custom-html.html

```diff
@@ -0,0 +1,15 @@
+<accordion>
+  <accordion-group #group>
```

Comment:
`#group` is not used and duplicated,
clean up things you don't need to have in demos
they should be simple and minimal
not confusing